### ISSUE TITLE
PyWeek: Collection of DNW info

### ISSUE THREAD

**Issue by einarf** [pythonarcade/arcade#1182]
Going through the PyWeek entires using arcade we collect some information on "did not work" votes on entires. This can help discovering compatibility issues.

* Some entires were using arcade 2.6.11 suffering from the gpu collision issues on some iGPUs

* Jeb reported issue with enums on initialization:
![bilde](https://user-images.githubusercontent.com/5418180/162606341-de701393-f854-42b6-b7fb-760984817a09.png)

* I'M THE EVIL TWIN: Several people had this problem. Probably related to hit box data.
```py
self.physics_engine_2_electric_boogaloo.update()
File "F:\pyweek\venv\lib\site-packages\arcade\physics_engines.py", line 245, in update
return _move_sprite(self.player_sprite, self.walls, ramp_up=False)
File "F:\pyweek\venv\lib\site-packages\arcade\physics_engines.py", line 146, in _move_sprite
collision_check = check_for_collision_with_lists(moving_sprite, walls)
File "F:\pyweek\venv\lib\site-packages\arcade\sprite_list\spatial_hash.py", line 388, in check_for_collision_with_lists
if sprite is not sprite2 and _check_for_collision(sprite, sprite2):
File "F:\pyweek\venv\lib\site-packages\arcade\sprite_list\spatial_hash.py", line 254, in _check_for_collision
return are_polygons_intersecting(
File "F:\pyweek\venv\lib\site-packages\arcade\geometry_python.py", line 53, in are_polygons_intersecting
if cast(float, max_a) <= cast(float, min_b) or cast(float, max_b) <= cast(float, min_a):
TypeError: '<=' not supported between instances of 'float' and 'NoneType'
```

**Comment by MrTanoshii** [comment]
The issue from Jeb strangely resembles the issue from another team member (screenshot below):

![](https://cdn.discordapp.com/attachments/953179134396358659/956469278142500895/unknown.png)

^ He was running an Intel Pentium B960
From https://en.wikipedia.org/wiki/List_of_Intel_Pentium_processors and https://en.wikipedia.org/wiki/Intel_Graphics_Technology#Capabilities_(GPU_hardware), I conclude that it only supports OpenGL 3.1 while `arcade` requires OpenGL 3.3+

Jeb was also a team member, I'll ask him to check his OpenGL and comment with the update

**Comment by einarf** [comment]
> Jeb was also a team member, I'll ask him to check his OpenGL and comment with the update

We have removed that enum in another branch, so arcade might work for him if he's lucky.

EDIT: It was removed from development branch now, so he can try to install that if he wants.

**Comment by einarf** [comment]
These should be fixed in 2.7. It might not mean that the game will work, but it will certainly help.
